### PR TITLE
Fix the Document API

### PR DESCRIPTION
**Pull request by jbevain**

### CODE REVIEW

File: Mono.Cecil.Cil/Document.cs [hunk 1]
```diff
@@ -62,23 +62,43 @@ public sealed class Document : DebugInformation {
...
+			get { return language_vendor.ToVendor (); }
+			set { language_vendor = value.ToGuid (); }
+		}
+
+		public Guid LanguageVendorGuid {
+			get { return language_vendor; }
+			set { language_vendor = value; }
 		}
 
 		public byte [] Hash {
```
Comment (tmat) [inline Mono.Cecil.Cil/Document.cs]: >public byte [] Hash { [](start = 2, length = 21)

Let's also add byte[] EmbeddedSource { get; set; }

Comment (jbevain) [reply Mono.Cecil.Cil/Document.cs]: @tmat yep. What's that about the has with the start and the length?

Comment (tmat) [reply Mono.Cecil.Cil/Document.cs]: I'm not sure I understand your question.

Comment (jbevain) [reply Mono.Cecil.Cil/Document.cs]: What does this mean?

tmat

Comment (tmat) [reply Mono.Cecil.Cil/Document.cs]: Oh, that's an artifact of CodeFlow PR review tool.

Comment (jbevain) [reply Mono.Cecil.Cil/Document.cs]: 🤣

Comment (tmat) [reply Mono.Cecil.Cil/Document.cs]: I can see how it's very confusing in this context.

**Pull request merged**